specification/monitor/resource-manager/readme.md

```diff
@@ -44,7 +53,7 @@ input-file:
 
 These settings apply only when `--tag=package-2020-03` is specified on the command line.
 
-```yaml $(tag) == 'package-2020-03'
+``` yaml $(tag) == 'package-2020-03'
```

Comment:
Could you remove these changes? Looks like it's a regression.

Reply:
same as [above](https://github.com/Azure/azure-rest-api-specs/pull/10101#discussion_r453459093)